lib/index.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const yaml = require('js-yaml');
+const nunjucks = require('nunjucks');
+
+nunjucks.configure('../templates', { autoescape: false });
+
+
+const NGINX_TEMPLATE_FILE = 'nginx.conf.nj';
+
+function getPageConfigs(configFile) {
+  const contents = yaml.safeLoad(fs.readFileSync(configFile, 'utf-8'));
+  const configs = contents.map((c) => {
+    if (typeof c === 'string') {
+      return { from: c, to: c };
+    }
+    if (!c.from || !c.to) {
+      throw new Error('Non-string configs must have "to" and "from" properties');
+    }
+    return c;
+  });
+  return configs;
```

Comment:
I'm not very concerned about this since the integration test will fail if anything weird goes in there. Or am I missing a potential problem that wouldn't be caught there?